test/JDBC/expected/babel_format_custom_date_time.out

```diff
@@ -257,7 +257,7 @@ select FORMAT(CAST('2018-01-01 14:00' AS datetime2), N'hh:mm t') ;
 GO
 ~~START~~
 nvarchar
-02:00 PM
+02:00 A
```

Comment:
Fixed - I am documenting the limitations here: https://quip-amazon.com/eUjeAxLtEPu5/FORMAT-Known-Limitations

Essentially, postgres' to_char doesn't support 1 letter meridian (like A instead of AM etc.) so opting to return the correct "AM/PM" in 2 letter form instead of single letter "A" regardless of the time.